jcl/src/java.management/share/classes/com/ibm/java/lang/management/internal/LoggingMXBeanImpl.java

```diff
@@ -49,8 +48,47 @@
 /*[ENDIF]*/
 			PlatformLoggingMXBean {
 
-	private static final LoggingMXBeanImpl instance = new LoggingMXBeanImpl();
-
+	private static final LoggingMXBeanImpl instance;
+	
+	private static final Class<?> logManagerClass;
+	private static final Class<?> loggerClass;
+	private static final Class<?> levelClass;
+	private static final Method lm_getLogManager;
+	private static final Method lm_getLogger;
+	private static final Method lm_getLoggerNames;
+	private static final Method lo_getLevel;
+	private static final Method lo_getParent;
+	private static final Method lo_getName;
+	private static final Method lo_setLevel;
+	private static final Method le_getName;
+	private static final Method le_parse;
+	private static final String lm_LOGGING_MXBEAN_NAME;
+	
+	static {
+		try {
+			logManagerClass = Class.forName("java.util.logging.LogManager");
```

Comment:
Shouldn't this be using the `Class.forName(Module, String)` api instead?  The `java.logging` module can be looked up using `ModuleLayer.boot().findModule("java.logging")`?